proxy/src/registry.rs

```diff
@@ -597,6 +643,8 @@ impl Client for Registry {
         id: Option<String>,
         fee: protocol::Balance,
     ) -> Result<Transaction, error::Error> {
+        // TODO(xla): Remove automatic prepayment once we have proper balances.
+        self.prepay_account(author.public(), 1000).await?;
```

Comment:
This is to make sure an author always has funds before registering their handle.

Reply:
Are these funds used to pay the fee? I was under the impression that the org pays the registration of its members.

Reply:
This prepayment happens when registering a user (not the org), so these funds are used for paying the transaction of registering the same authors user handle. Member registration is payed by the org.
Since we need several registered users for registering a new member, we need to give users other than alice funds automatically. As explained in the Note of the description of the PR, this only helps for running cargo test as we can bypass the api directly. The api and thus the playground always uses alice as the author for every transaction